src/jsutils/Path.js

```diff
@@ -7,6 +7,8 @@ export type Path = {|
 
 /**
  * Given a Path and a key, return a new Path containing the new key.
+ *
+ * @internal
```

Comment:
Same goes for `jsutils` folder:
https://github.com/graphql/graphql-js/blob/master/src/jsutils/README.md
> These functions are not part of the module interface and are subject to change.

Reply:
Sure, I can change that :+1: 

It's maybe worth noting that in this same file, `pathToArray` is exported as public in `src/index.js` (aliased as `responsePathAsArray`).

https://github.com/graphql/graphql-js/blob/83d5011df8d5fc466d08517a2d13f9d3fb527161/src/index.js#L287

Reply:
@craicoverflow Good catch 👍 I need to think about it 🤔 💭